build.gradle.kts

```diff
@@ -38,9 +38,10 @@ tasks {
 }
 
 dependencies {
-  // logging
+  // logging + telemetry
   implementation("net.logstash.logback:logstash-logback-encoder:6.6")
   implementation("ch.qos.logback:logback-classic:1.2.3")
+  implementation("com.microsoft.azure:applicationinsights-agent:3.0.2")
```

Comment:
Is this not already brought in by the dps spring boot plugin? 
https://github.com/ministryofjustice/dps-gradle-spring-boot/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/gradle/configmanagers/AppInsightsConfigManager.kt

Reply:
yeh good spot thanks